docs/getting_started/intellij_plugin.md

```diff
@@ -0,0 +1,96 @@
+
+Examples:
+=== "target Completion"
+    ![target Completion](../img/idea_plugin/target_completion.png)
+
+=== "Missing required attribute"
+    ![required attribute](../img/idea_plugin/required_attribute.png)
+
+### Execution
+=== "from Kotlin"
+    Function annotated with `@KChutney` can be executed and synchronized
+    ![run from Kotlin](../img/idea_plugin/run_kotlin.png)
+
+=== "from json"
+    You can run your scenario from the json file by clicking on:
+
+   * run icon. Scenario will be executed on the embedded Chutney Instance.
+     The embedded server will run on a toolWindow where you can see the logs
+      ![embedded Chutney](../img/idea_plugin/local_server.png)
+   * run menu item in the contextual menu when right-click:
+     * in scenario json file.
+       ![run scenario](../img/idea_plugin/run_scenario.png)
+     * on a folder.
+       ![run all](../img/idea_plugin/run_scenarios.png)
+  
+     You can also define a run configuration and choose if execution will be done on the embedded or the remote Chutney instance.
+
+### Report
+In Services view, you can run, stop and view execution report
+![execution report](../img/idea_plugin/exec_report.gif)![](../img/idea_plugin/run_kotlin.png)
+
+### Synchronize with remote Chutney
+!!! note
+    In order to get access to a remote Chutney instance, some properties are required. </br>
+    Go to File -> Settings -> tools -> Chutney and fill-in your Chutney remote instance properties
+    ![](../img/idea_plugin/remote_conf.png)
+
+Synchronization actions are available in right-click contextual menu.
+![sync actions](../img/idea_plugin/sync_actions.png)
+
+* Update remote scenario from local
+  Push current file content to remote Chutney in order to override scenario content.
+* Update local scenario from remote server
+  Pull remote scenario content to local file.  
+
+* Add scenario from remote server
+    Create scenario on the remote Chutney.
+
+* Show diff between local scenario file and remote
+  Compare local and remote scenario content.
+  ![scenario diff](../img/idea_plugin/scenario_diff.png)
+
+* Editor Notifications
+  When you open a Chutney Scenario File and if the scenario is different from the remote version, a notification with action is shown as below
+ ![diff notification](../img/idea_plugin/diff_notif.png)
+
+* Open remote scenario file in browser
+  Open remote scenario's last execution in the default browser
+
+* Add scenario to an existing campaign
+  ![add to campaign](../img/idea_plugin/add_to_campaign.png)
+
+* Get All scenarios of a campaign
+  On an empty directory call the action Get scenarios from Campaign.
+  ![get campaign scenarios](../img/idea_plugin/get_campaign_scenarios.png)
+
+### Convert from HJSON to JSON
+If you copy/paste HJSON content, the IDE will automatically transform it to json.
```

Comment:
Maybe link to HJSON syntax external link ?
Why is this feature exists ? Maybe we have to explain the reason behind this ?